### DISCUSSION TITLE
Support image resize

### DISCUSSION THREAD

**Discussion by zoni** [zoni/obsidian-export#59] [Feature requests]
Obsidian's image resize syntax is currently unsupported. This is a tracking issue for support of this functionality.

For reference, the syntax for this in Obsidian is as follows:

- Markdown images: `![AltText|100x100](https://url/to/image.png)`
- Embeds: `![[image.png|100x100]]`

(See _Resize images_ on https://publish.obsidian.md/help/How+to/Embed+files)

**Comment by epsilonhalbe** [comment]
hey zoni,

i assume this feature is still missing.
any tips before I get started on tackling it
